### ISSUE TITLE
tqdm.auto does not work fine in Atom hydrogen

### ISSUE THREAD

**Issue by kamione** [tqdm/tqdm#1159]
## Summary

tqdm.auto progress bar does not update if the codes are running in a Atom IDE with hydrogen

## Additional details

- I run the NiMARE package which it uses tqdm.auto
- NiMARE version: 0.0.7
- I have reported this issue in the [the NiMARE repo](https://github.com/neurostuff/NiMARE/issues/481) but this issue is generally caused by tqdm.auto in Atom Hydrogen environment [

### What were you trying to do?
I tried to run:
```python
ale = nimare.meta.cbma.ALE()
res = ale.fit(dset)

corr = nimare.correct.FWECorrector(method="montecarlo", n_iters=10, n_cores=-1)
cres = corr.transform(ale.results)
```

### What actually happened?
This block of codes only produced:
```
INFO:nimare.correct:Using correction method implemented in Estimator: nimare.meta.cbma.ale.ALE.correct_fwe_montecarlo.
  0%|          | 0/10 [00:00<?, ?it/s]
INFO:nimare.meta.cbma.base:Using null distribution for voxel-level FWE correction.
```
The codes worked perfectly but the progress bar does not update if I work in a Atom IDE with hydrogen
## Reproducing the bug

```python
# This problem can be simply reproduced in a Atom IDE with hydrogen by:
import time
from tqdm.auto import tqdm

for _ in tqdm(range(10)):
    time.sleep(1)
    pass
```

**Comment by casperdcl** [comment]
Yes this is essentially an old issue with the design of the IPython kernel itself (see the accepted albeit relatively unpopular answer at https://stackoverflow.com/a/15412661/3896283).

You could replace `tqdm.auto.tqdm` with simply `tqdm.tqdm`, or perhaps try to implement one of the alternative work-arounds/hacks on that SO page.
